xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Title.xml

```diff
@@ -45,14 +45,14 @@
     #set ($value = $xwiki.getDocument("$stringtool.removeEnd($className, 'Class')Template").title)
   #else
     ## We are editing an application entry so the title must be read from / written to the current document.
-    #set ($name = 'title')
+    #set ($name = 'title1')
     #set ($value = $tdoc.title)
     #if ("$!value" == '')
       #set ($value = $tdoc.documentReference.name)
     #end
   #end
   {{html clean="false"}}
-  &lt;input type="text" name="$name" value="$!escapetool.xml($value)"
+  &lt;input id="${prefix}$name" type="text" name="$name" value="$!escapetool.xml($value)"
```

Comment:
This is missing escaping.

Reply:
Do we need to escape all variables declared locally? This $name, just like the $prefix, is defined a few lines above and I didn't think it was necessary to escape it. 
I added escaping on the name in https://github.com/xwiki/xwiki-platform/pull/3584/commits/e805384e8881fd0e7612f6ad4b15ed70ea209b48 nevertheless :+1: